packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.tsx

```diff
@@ -45,6 +45,8 @@ export type MarkdownSynapseProps = {
   onMarkdownProcessingDone?: (textContent: string | null | undefined) => void
   showPlaceholderIfNoWikiContent?: boolean
 }
+type MarkdownSynapseComponent = React.ComponentType<MarkdownSynapseProps>
```

Comment:
Use this type to define `MarkdownSynapse` instead of casting it at the end. This type lets us easily create a type-safe mock that is a function component. Without this type, the mock would be required to be a class component.